scripts/convert-to-c.js

```diff
@@ -3,7 +3,10 @@
 console.log('-- loading: convertToC');
 
 
-function convertToC() { }
+function convertToC(fahrenheit) {
+  const celsius = (fahrenheit - 32) * 5/9;
```

Comment:
Maybe Math. round() could be used to return a rounded result? Cause now -14 returns  6.800000000000001

Reply:
Yes I  sow it. the problem was in the function for the convertToCHandler had some spelling mistake.  Fixing it now.

Reply:
It is done.